app/src/main/java/net/kwatts/powtools/util/SharedPreferencesUtil.java

```diff
@@ -101,4 +120,41 @@ public void saveMacAddress(String macAdress, String macAddressName) {
     public boolean isMetric() {
         return mSharedPref.getBoolean(METRIC_UNITS, false);
     }
+
+    public float getSpeedAlert() {
+
+        boolean isMetric = App.INSTANCE.getSharedPreferences().isMetric();
```

Comment:
`isMetric()` is a method of this class, no need to use App instance.

Could inline the `isMetric()` call as well.